tests/unit-tests/#.scm

```diff
@@ -123,10 +126,22 @@
                          (##sourcify tail-exn? src))
                    src)))))
 
+           (define (##record-tested-procedures! form)
```

Comment:
The method for extracting the procedure that is being tested is rather brittle.  This should be improved.